### PR TITLE
London-class 7 week 2(git)  (Ali Nosratipour)  amended answers

### PR DESCRIPTION
**Pull request by alinosratipour**
questions number 1-16 I have corrected the answers.

questions 14,17,18 I adedd some more details to the answers.

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -1,113 +1,178 @@
...
 When people want to show off code to each other they can put it on Github
 
+correct answer is:
+
+Documentation.
+Showcase your work.
+Markdown.  
+Track changes in your code across versions.
+Integration options.
+Collaboration
```
Comment (Trobinsonshoots) [inline homework.md:85]: great answers, maybe you can cut down to just three.

File: homework.md [hunk 2]
```diff
@@ -1,113 +1,178 @@
...
+
 ## Question 14
 
 What is Github?
 
 A place for us to store and share our code.
 
+correct answer is:
+
+A website to create and hold repository and it will sync with git software
```
Comment (NeginDn) [inline homework.md:143]: Hi Ali, I don't think github Sync with git necessarily

**Review by NeginDn — CHANGES_REQUESTED** [review]
Questions were answered very well, well done. Just a suggestion, I don't think you need wrong anseres there, maybe deleting them and replacing them with the right ones would look better?

File: homework.md (hunk 2 above)
Comment (alinosratipour) [reply homework.md:143]: I'm not sure but I think make it sync according to this link but I will ask it from teacher to make sure https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/keeping-your-local-repository-in-sync-with-github

**Pull request closed without merging**